### ISSUE TITLE
README ampersands are encoded

### ISSUE THREAD

**Issue by ashleygwilliams** [npm/newww#1789]
moving from: https://github.com/npm/marky-markdown/issues/124 cc/ @lewiscowper @revin

![099e6090-c10a-11e5-8587-c67a956d6c20](https://cloud.githubusercontent.com/assets/1163554/12515062/5b3487d8-c0f4-11e5-898e-d2121014ae62.png)

**Comment by revin** [comment]
In the marky version of this issue, I suggested `{{{` would fix this, but hmm. Reading #1730, looks like the change from `{{{` to `{{` was intentional for package descriptions?

**Comment by ashleygwilliams** [comment]
yes, i am going to talk about https://github.com/npm/newww/pull/1731 with @aredridel today

**Comment by lewiscowper** [comment]
Not limited to ampersands, also inline code blocks, as seen on https://www.npmjs.com/package/in-publish

![screen shot 2016-01-25 at 18 14 02](https://cloud.githubusercontent.com/assets/593574/12558376/7b275faa-c38f-11e5-8ede-f8fad5900dc2.png)

**Comment by revin** [comment]
Hi all, got another one. @rnjailamba reported the same encoded ampersand issue on the package description for [urlgenerator](https://www.npmjs.com/package/urlgenerator) (via [npm/marky-markdown#167](https://github.com/npm/marky-markdown/issues/167):

![screen shot 2016-03-18 at mar 18 7 25 41 am](https://cloud.githubusercontent.com/assets/94850/13880685/ac608906-ecda-11e5-95ca-8874a9d3a562.png)

**Comment by rnjailamba** [comment]
@revin @aredridel  any progress on this?

**Comment by donaldpipowitch** [comment]
Another example for `"` to `&quot;`: https://www.npmjs.com/package/rfnd-hello-world.

**Comment by addaleax** [comment]
Yeah, I think #1731 is not the best solution for the problem it sought to fix; in particular, the description is explicitly parsed as markdown [here](https://github.com/npm/newww/blob/db1f3dbb183f43c886da6eb25ae214370649c1e2/presenters/package.js#L211-L234), and that makes little sense if the result will be escaped as HTML.

Even the description of the package which prompted the original bug report [react-contenteditable](https://www.npmjs.com/package/react-contenteditable) is still incorrectly rendered, because it’s still interpreted as markdown (note the extra `</div>` in the end):
![react-contenteditable-head](https://cloud.githubusercontent.com/assets/899444/15804392/db501d2a-2b06-11e6-997e-7129f0b88aa1.png)

Since many packages* have come to rely on being able to use markdown in their descriptions (and `npm init` even extracts the description text from a README.md if present), I’d suggest reverting #1731, or, maybe (big maybe?) adding some kind of rough format auto-detection.

(*) A quick automated sample showed about 1 % of packages having backticks for code in their description… not as much as I subjectively would have guessed, but still.
